File: app/src/main/java/ms/imagine/foodiemate/api/Prediction.java
package ms.imagine.foodiemate.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Prediction {
    @SerializedName("TagId")
    @Expose
    public String tagId;

    @SerializedName("Tag")
    @Expose
    public String tag;

    @SerializedName("Probability")
    @Expose
    public Double probability;

}

File: app/src/main/java/ms/imagine/foodiemate/api/Prediction.kt
package ms.imagine.foodiemate.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Prediction {
    @SerializedName("TagId")
    @Expose
    var tagId: String? = null
    @SerializedName("Tag")
    @Expose
    var tag: String? = null
    @SerializedName("Probability")
    @Expose
    var probability: Double? = null

}